### PR TITLE
Refactor sample apps

### PR DESCRIPTION
**Pull request by aslo**
This PR combines the sample apps into a single Spring Boot app that can be deployed into either of the `openjdk` or the `jetty` runtimes. Separate maven profiles are used to control how the app is packaged.

I followed these [spring boot docs](https://spring.io/blog/2014/03/07/deploying-spring-boot-applications#what-about-the-java-ee-application-server) to set up the project.

### CODE REVIEW

File: fixtures/sample-app/pom.xml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    <version>1.0.0-SNAPSHOT</version>
+    <packaging>${project.packaging}</packaging>
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>1.4.0.RELEASE</version>
+    </parent>
+
+    <dependencies>
+	<dependency>
```
Comment (meltsufin) [inline fixtures/sample-app/pom.xml]: You're mixing tabs an spaces in this file. Use 2 spaces for a tab.

File: test/debugger.sh [hunk 2]
```diff
@@ -5,25 +5,19 @@
...
-declare -a test_apps=($fixtures_dir/jar-app $fixtures_dir/servlet-app)
+sample_app="$project_root/fixtures/sample-app"
+declare -a runtimes=("openjdk" "jetty")
 
 source $project_root/helpers.sh
 
+# TODO grep this from deploy output
 project=`get_active_project`
 project_url_suffix="$(echo $project | tr _ -).appspot.com"
 project_version="debugger-test"
```
Comment (meltsufin) [inline test/debugger.sh:16]: This may be more appropriate for a separate PR, but maybe we should use $runtime as the project_version, and also we should delete the version after the test is done - gcloud versions delete.

Comment (aslo) [reply test/debugger.sh:16]: That's a good idea. I can tackle that in another PR.

File: fixtures/sample-app/pom.xml (hunk 1 above)
Comment (aslo) [reply fixtures/sample-app/pom.xml]: Fixed

File: test/debugger.sh (hunk 2 above)
Comment (meltsufin) [reply test/debugger.sh:16]: Created issue: #4
